models/_sector/dex/trades/arbitrum/_schema.yml

```diff
@@ -2,6 +2,8 @@ version: 2
 
 models:
   - name: dex_arbitrum_base_trades
+    tests:
+      - check_dex_info_relationship
```

Comment:
Do you want to use the `relationships` test here?
It may be one test file less.

But either way works for me.

Reply:
I don't mind either (and here comes 😉) but:

1. we're using kinda similar approach of adding more tests with custom seed check macros
2. we'll have 1 test file less, but more lines in each `_schema.yml` file:

with macro:
```
  - name: dex_arbitrum_base_trades
    tests:
      - check_dex_info_relationship
```

with built-in `relationships` test:
```
  - name: dex_arbitrum_base_trades
    columns:
      - name: project
        tests:
          - relationships:
              to: ref('dex_info')
              field: project
```

Reply:
Convinced me.